my-portfolio/src/components/cards.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+export const Card = ({ title, description, id, icon }) => {
+  return (
+    <Link to={`/projects/${id}`}>
+      <div className="cardContainer">
+        {icon && <img className="cardIcon" alt="icon" src={icon}></img>}
+        {title && <h2 className="cardTitle">{title}</h2>}
+        {description && <p className="cardDescription">{description}</p>}
```

Comment:
A good practice is to have a fallback of each of theses:

```suggestion
        <img className="cardIcon" alt="icon" src={icon || fallbackIcon }></img>
        <h2 className="cardTitle">{title || fallbackTitle}</h2>
        <p className="cardDescription">{description || fallbackDescription}</p>
```